CONTRIBUTING_DOCS.md

```diff
@@ -132,27 +132,73 @@ Examples:
 - Calico API schema reference (policy, ip pool, etcd)
 
 
-## Doc site architecture
+## Organizational changes
 
-The naming and layout of these navbars are stored in `_data/$VERSION/navbars/*`. Jekyll automatically stores information from the `_data` directory in an accessible variable called `site.data`. The top-level layout (`_layout/docwithnav.html`) will iterate through all the files in `site.data[version].navbars` to construct the sidebar based on which version is being viewed.
+### Creating new pages
+
+- To create a top level splash page for a URL path, simply name the file `index.md`.
+
+- If you are adding a new page to the current release directory, add the following metadata just under the `title` of the page: `redirect_from: latest/<path-to-new-file>`. This ensures that readers who use `latest` in their paths instead of the version number of the current release get redirected to your new page. If you do not add this, you may experience `htmlproofer` errors.
+
+- [Add the new page to the side navigation bar](#linking-content).
+
+- Within the copies of the page in the `master` and previous release directories, add a `canonical_url` line below the `title` line in the metadata of the page. This should contain the absolute path to the page in the current latest directory. Example: `canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/'`
+
+
+### Deleting or renaming pages
```

Comment:
This section is great, but we're still missing an explanation of how this canonical URL is supposed to work. from what I've gathered here:

"Each version of a page should include in its metadata a `canonical_url` which points to the latest version of the page."

I'd like it if we could include that simple description somewhere in the contributing docs considering we have managed to incorrectly implement canonical URLs once, and it helps spread the knowledge of how it works.